src/OpenCensus.Exporter.Prometheus/Middleware/PrometheusExporterMiddlewareExtensions.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="PrometheusExporterMiddlewareExtensions.cs" company="OpenCensus Authors">
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenCensus.Exporter.Prometheus.Middleware
```

Comment:
it's not a bad idea to use the namespace 
`namespace Microsoft.AspNetCore.Hosting` so the extension will be visible without the need to import namespace first.